src/components/ui/menu/menuGroup.component.tsx

```diff
@@ -25,6 +25,7 @@ import { MenuItemDescriptor } from './menu.service';
 
 export interface MenuGroupProps extends MenuItemProps {
   children?: ChildrenWithProps<MenuItemProps>;
+  expanded?: boolean;
```

Comment:
Confusing naming. I'd say we better go with `initialExpanded` since you aren't able to control the component state after its initial render.